misc/specs/atrium-specs-jvm/src/main/kotlin/ch/tutteli/atrium/specs/integration/OptionalAssertionsSpec.kt

```diff
@@ -27,12 +28,12 @@ abstract class OptionalAssertionsSpec (
         val isEmptyFun = isEmpty.lambda
```

Comment:
I suggest you add the isPresent.name to this `describeFun` (as second argument).